app/assets/src/components/views/discovery/DiscoverySidebar.jsx

```diff
@@ -163,7 +172,13 @@ export default class DiscoverySidebar extends React.Component {
         {extraRows.length > 0 && (
           <div
             className={cs.showHide}
-            onClick={() => this.toggleExpandedMetadataGroup(field)}
+            onClick={() => {
+              this.toggleExpandedMetadataGroup(field);
+              logAnalyticsEvent("DiscoverySidebar_show-more-toggle_clicked", {
```

Comment:
for the sake of analytics, would you recommend breaking "toggle" functions into a show function and a hide function?

Reply:
We should just make sure the info of showing or hiding is available in the logged event, which it was not here, so good catch!